scripts/ci/build-docs.sh

```diff
@@ -22,7 +22,9 @@ rm -rf gh-pages/deployed*
 cmake -DCHFL_BUILD_DOCUMENTATION=ON .
 cmake --build . --target doc_html
 rm -rf doc/html/.doctrees/ doc/html/.buildinfo
-rm -rf doc/html/_static/bootswatch-* doc/html/_static/bootstrap-2.3.2/
+rm -rf doc/html/_static/bootswatch-2.3.2/ doc/html/_static/bootstrap-2.3.2/
```

Comment:
I'm not sure why this is necessary at first glance. Mind explaining it to me?

Reply:
The documentation (http://chemfiles.org/chemfiles/latest/) is missing `boostrap.min.css` which is fetched from `doc/html/_static/bootswatch-3.7.3/flatly/bootstrap.min.css`